lib/Elastica/Aggregation/BucketSelector.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Elastica\Aggregation;
+
+use Elastica\Exception\InvalidException;
+
+/**
+ * Class BucketSelector.
+ * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-bucket-selector-aggregation.html
+ */
+class BucketSelector extends AbstractSimpleAggregation
+{
+    /**
+     * @param string      $name
+     * @param array|null  $bucketsPath
+     * @param string|null $script
+     */
+    public function __construct(string $name, string $bucketsPath = null, string $script = null)
```

Comment:
here the 2nd parameter is array

Reply:
I'm sorry, this happens when I do changes like that on the train :(

Reply:
ahahahah no problem ! luckily CI saves us from this